src/components/WPResponsiveImage/WPResponsiveImage.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Image } from '..';
+import { arrayFromRange, resizeWPImage } from '@quartz/js-utils';
+
+function WPResponsiveImage( {
+	alt,
+	aspectRatio,
```

Comment:
i tend to favor passing in width and height and calculating aspectRatio internally, esp. since we generally get width and height from our APIs. with an aspect ratio prop, 99% of the time we'd be doing this:

width={width}
aspectRatio={width / height}

Reply:
I know what you're saying, but in this case we aren't supplying a set width and height, just a range of widths. I could make fallbackHeight required and determine the aspect ratio from that, but I don't know if that's as clear to the developer. WDYT?

Reply:
requiring both `fallbackWidth` and fallbackHeight` is clearest for me, since that parallels `<img>` attributes. but i don't feel super strongly